cmd/cert-checker/main.go

```diff
@@ -411,13 +412,15 @@ func (c *certChecker) checkCert(ctx context.Context, cert core.Certificate, igno
 		if features.Enabled(features.CertCheckerRequiresCorrespondence) {
 			precertDER, err := c.getPrecert(ctx, cert.Serial)
 			if err != nil {
-				problems = append(problems,
-					fmt.Sprintf("fetching linting precertificate for %s: %s", cert.Serial, err))
+				// Log and continue, since we want the problems slice to only contains
+				// problems with the cert itself.
+				c.logger.Errf("fetching linting precertificate for %s: %s", cert.Serial, err)
+				atomic.AddInt64(&c.issuedReport.DbErrs, 1)
 			} else {
 				err = precert.Correspond(precertDER, cert.DER)
 				if err != nil {
 					problems = append(problems,
-						fmt.Sprintf("checking correspondence for %s: %s", cert.Serial, err))
+						fmt.Sprintf("Certificate does not correspond to precert for %s: %s", cert.Serial, err))
```

Comment:
```suggestion
						fmt.Sprintf("certificate does not correspond to precert for %s: %s", cert.Serial, err))
```

Reply:
For better or worse, all of the other strings which get appended together into the list of problems start with a capital letter. See lines 389 and 394, for example.